src/scss/components/common/Input.scss

```diff
@@ -1,4 +1,4 @@
-.manual-input, input[type="text"], input[type="password"] {
+.manual-input { //input[type="text"]:not(.field), input[type="password"]:not(.field)
   // display: inline-block;
```

Comment:
remove comment

Reply:
This comment was nessesary. I forgot write it. This global styles overwriting styles written in components. Thrownin comment because of
possibility to come back